app/ReadmeFormatter.php

```diff
@@ -74,13 +81,13 @@ public function replaceImageUrlsHtml($html)
 
     public function replaceNonImageUrlsHtml($html)
     {
-        $pattern = '/<a href="((?!http)(?!mailto).*?)"/i';
+        $pattern = '/<a(.+)href="((?!http)(?!mailto).*?)"/i';
```

Comment:
Also a lazy quantifier here to prevent over-matching. Example: https://regex101.com/r/jFUceC/2
```suggestion
        $pattern = '/<a(.+?)href="((?!http)(?!mailto).*?)"/i';
```